tests/hyperparamopt/test_hpo.py

```diff
@@ -0,0 +1,89 @@
+#  Copyright 2016 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+import pytest
+
+
+def test_simple_gmm():
+    from brainiak.hyperparamopt.hpo import gmm_1d_distribution
```

Comment:
Do we have a convention for doing imports only in the method body for tests? If not, I'd say move all the imports to the top of the file like a normal module.